### PR TITLE
Issue 362 dialect selector test

### PR DESCRIPTION
**Pull request by shannonlal**
This is a continuation of the work on Issue #362.  This is some basic unit (Jest) tests for the Dialect Selector.  This was a very simple component and really could only think of two tests.  I welcome additional comments and feed back on this.

### CODE REVIEW

**Comment by shannonlal** [comment]
@n-riesco If you have a few minutes tomorrow would you mind having a look at this PR.  I tried to follow your previous comments on the Connect Button.  I welcome your comments.  Thanks in advance.

File: test/app/components/Settings/DialectSelector/DialectSelector.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            updateConnection={updateConnection}
+        />);
+
+        // Logos and Images should be greater than 1
+        expect(selector.find('.logo').length).toBeGreaterThan(1);
+        expect(selector.find('.logoImage').length).toBeGreaterThan(1);
+
+        // Number of Logos should match number of images
+        expect(selector.find('.logo').length).toEqual(selector.find('.logoImage').length);
+
```
Comment (n-riesco) [inline test/app/components/Settings/DialectSelector/DialectSelector.test.js]:
```
import {DIALECTS} from `../../../../../app/constants/constant.js`
expect(selector.find('.logo').length).toEqual(DIALECTS.length);
```

File: test/app/components/Settings/DialectSelector/DialectSelector.test.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        const connectionObject = {
+        };
+
+        const selector = mount(<DialectSelector
+            connectionObject={connectionObject}
+            updateConnection={updateConnection}
+        />);
+
+        // Logos and Images should be greater than 1
+        expect(selector.find('.logo').length).toBeGreaterThan(1);
```
Comment (n-riesco) [inline test/app/components/Settings/DialectSelector/DialectSelector.test.js]: :scissors: this is tested below

File: test/app/components/Settings/DialectSelector/DialectSelector.test.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        expect(selector.find('.logoImage').length).toBeGreaterThan(1);
```
Comment (n-riesco) [inline test/app/components/Settings/DialectSelector/DialectSelector.test.js]: :scissors: this is tested below

File: test/app/components/Settings/DialectSelector/DialectSelector.test.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            dialect: 'mysql'
+        };
+
+        const selector = mount(<DialectSelector
+            connectionObject={connectionObject}
+            updateConnection={updateConnection}
+        />);
+
+        // Dialect not selected so should not be found
+        expect(selector.find('.logoSelected').length).toBe(1);
```
Comment (n-riesco) [inline test/app/components/Settings/DialectSelector/DialectSelector.test.js]: `expect(selector.find('.logoSelected').first().key()).toBe(connectionObject.dialect);`

**Comment by n-riesco** [comment]
:dancer:

**Pull request merged**